docs/graphql-api/subscriptions/index.mdx

```diff
@@ -0,0 +1,191 @@
+ddn codemod enable-model-subscriptions
+```
+
+This command updates the local metadata files, adding subscription-specific configurations. After running this, create a
+supergraph build and test the subscriptions on the API. For futher customization, you can manually
+[update the metadata](#manual-metadata-configuration).
+
+### Manual Metadata Configuration
+
+You can manually configure subscriptions by editing the metadata HML files in your DDN project.
+
+#### Root Type Name
+
+Set or update the subscription root type in your GraphQL API by modifying the `graphql-config.hml` file located at
+`globals/metadata/graphql-config.hml`. This ensures the subscription root type is defined, which is necessary for
+enabling subscriptions. You will have to add the following in the `definition` for
+[`GraphqlConfig`](/supergraph-modeling/graphql-config.mdx#graphqlconfig-graphqlconfig)
+
+```yaml
+subscription:
+  rootOperationTypeName: Subscription
+```
+
+Example:
+
+```yaml {8-9} title="<project-root>/globals/metadata/graphql-config.hml"
+kind: GraphqlConfig
+version: v1
+definition:
+  query:
+    rootOperationTypeName: Query
+  mutation:
+    rootOperationTypeName: Mutation
+  subscription:
+    rootOperationTypeName: Subscription
+```
+
+#### Model Root Field
+
+For each model, update the root field name for subscriptions in the corresponding model's HML file. You can also
+configure other optional parameters, such as the [polling interval](#polling-interval), the root field's description,
+and its deprecation status.
+
+An example of an `Article` [model definition](/supergraph-modeling/models.mdx#model-model) with subscriptions enabled
+for the `selectMany`, `selectUniques`, and `aggregate` root fields:
+
+```yaml {13-15,20-24,29-30} title="<project-root>/<subgraph-name>/metadata/Article.hml"
+kind: Model
+version: v1
+definition:
+  name: Article
+  objectType: Article
+  source:
+    dataConnectorName: my_connector
+    collection: article
+  aggregateExpression: ArticleAggExp
+  graphql:
+    selectMany:
+      queryRootField: article
+      subscription:
+        rootField: article
+        description: Subscribe to article model
+    selectUniques:
+      - queryRootField: articleById
+        uniqueIdentifier:
+          - id
+        subscription:
+          rootFIeld: articleById
+          description: Subscribe to article model by its unique id
+          deprecated:
+            reason: Use article root field with filters
+    orderByExpressionType: ArticleOrderBy
+    filterInputTypeName: ArticleFilterInput
+    aggregate:
+      queryRootField: articleAggregate
+      subscription:
+        rootField: articleAggregate
+```
+
+#### Polling Interval
+
+You can set the polling interval (in milliseconds) within the model's
+[GraphQL definition](/supergraph-modeling/models.mdx#model-modelgraphqldefinition) to determine how often the engine
+checks for updates. If not specified, the interval defaults to `1000` milliseconds (1 second).
```

Comment:
Let's add one/two sentences on what are the real world use-cases for too long, too short or the default setting.